test/buffer.js

```diff
@@ -3,6 +3,7 @@ var test = require('tap').test;
 var vm = require('vm');
 
 if (!ArrayBuffer.isView) ArrayBuffer.isView = function () { return false; };
+if (!Object.setPrototypeOf) Object.setPrototypeOf = require('setprototypeof');
 
```

Comment:
instead of polyfilling it, should it be imported in the implementation? otherwise tests will be assuming the user has polyfilled it, which is not guaranteed.

Reply:
I think we can import it in the files that are being bundled for these tests, yes. `buffer` itself probably wouldn't want to add it because all of the browsers that they support have `setPrototypeOf`.